src/main/java/org/sagebionetworks/web/client/jsinterop/react/HasStyle.java

```diff
@@ -3,18 +3,21 @@
 import jsinterop.base.JsPropertyMap;
 import org.sagebionetworks.web.client.jsinterop.JsObject;
 import org.sagebionetworks.web.client.jsinterop.PropsWithStyle;
-import org.sagebionetworks.web.client.widget.ReactComponent;
+import org.sagebionetworks.web.client.jsinterop.ReactComponentType;
+import org.sagebionetworks.web.client.widget.ReactComponentV2;
 
 /**
  * Abstract class for React component widgets that have a style prop. The style prop for the component may be manipulated
  * to show/hide the component based on the current state of the widget.
  * @param <T> the prop type.
  */
-public abstract class HasStyle<T extends PropsWithStyle>
-  extends ReactComponent<T> {
+public abstract class HasStyle<
```

Comment:
Re-add HasStyle. No changes since original PR, other than updating the constructor (because the `super` changed)